src/routes.py

```diff
@@ -9,5 +12,12 @@
 @bp.route("/")
 @bp.route("/index")
 def index() -> str:  # pylint: disable=unused-variable
-    """Use this view function to check whether Flask is installed properly."""
-    return "Hello, wordl!"
+    """Use this functon to render index page's template."""
+    return render_template("index.html")
+
+
+@bp.route("/registration")
+def registration() -> str:  # pylint: disable=unused-variable
+    """Use this functon to render registration page's template."""
```

Comment:
Yep, now this handler is used only to render a form, but in the future in will probably do more things. So let's put in docstring something like:
```python
"""Handle user's registration form."""
```